themes/phudj/phudj.theme.bash

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+  #####################################
+ #  https://github.com/phreakocious  #
+#####################################
+
+# ★ ➤ ▶ ► ‣ ✱ ✽ ❖ ✸ ✹ ◼■ ●○•°✪ ⬤ ⚫⭕ ⧫⬧♦⬥
+
+      prompt_user=▶
+     prompt_clean=✔
+     prompt_dirty=✗
+     prompt_ahead=↑
+    prompt_behind=↓
+   prompt_diverge=↕
+
+      color_user="$bold_green"
+      color_host="$green"
+      color_time="\[$(echo_color rgb 265 265 265)\]"
+        color_at="$color_time"
+      color_path="$cyan"
+     color_clean="$bold_green"
+     color_dirty="\[$(echo_color rgb 255 95 0)\]"
+     color_ahead="$bold_yellow"
+    color_behind="$yellow"
+   color_diverge="$bold_white"
+    color_branch="\[$(echo_color rgb 0 255 95)\]"
+   color_success="$bold_green"
+   color_failure="$bold_red"
+
+prompt_user_root=★
+ color_user_root="$bold_red"
+ color_host_root="$red"
+
+
+###
+
+
+[[ $EUID -eq 0 ]] && color_host=${color_host_root} && color_user=${color_user_root} && prompt_user="$prompt_user_root"
+
+parse_git() {
+  git_status="$(git status 2>/dev/null)" || return
```

Comment:
Parsing `git status` output is generally discouraged, since the output may change between Git versions, and the text output can differ based on the configured language of the user. The patterns you're using below will only work for English language settings, for example.

In general, you should use `git status --porcelain` for things like this...